v1.1/debug-and-error-logs.md

```diff
@@ -49,12 +49,20 @@ Based on the command's flags and the message's [severity level](#severity-levels
 
 #### Write to File
 
-CockroachDB can write messages to log files, which use the following format:
+CockroachDB can write messages to log files.  The files are named using the following format:
 
 ~~~
 cockroach.[host].[user].[start timestamp in UTC].[process ID].log
 ~~~
 
+For example:
+
+~~~
+cockroach.richards-mbp.rloveland.2018-03-15T15_24_10Z.024338.log
+~~~
+
+{{site.data.alerts.callout_info}}All log file timestamps are in UTC because CockroachDB is designed to be deployed in a distributed cluster.  Nodes may be located in different time zones, and using UTC makes it easy to correlate log messages from those nodes no matter where they are located.{{site.data.alerts.end}}
```

Comment:
nit: I know it's your preference to use 2 spaces following a period, but throughout the docs we use only one space.

Reply:
Sorry, fixed in 102e434c

Adding some checks to my process so this stops slipping into my work writing.